src/util/generatePreview.js

```diff
@@ -38,6 +39,19 @@ const errorHandlerScript = `(${(() => {
   };
 }).toString()}());`;
 
+const alertReplacementScript = `(${(() => {
+  // Object.create so our reference
+  // doesn't get destroyed when we delete it on the window
+  const _swal = Object.create({swal: window.swal});
+
+  Reflect.defineProperty(window, 'alert', {
```

Comment:
K, was just blindly following the linter, I'd never even heard of Reflect until it yelled at me.